lms/static/coffee/src/instructor_dashboard/util.coffee

```diff
@@ -141,18 +140,20 @@ p_wrapper = (value) ->
     edx.HtmlUtils.HTML('</p>'),
   )
 
-unknown_p = p_wrapper(gettext('Unknown'))
+unknown_p = () ->
+  p_wrapper(gettext('Unknown'))
+
```

Comment:
@andy-armstrong @bjacobel This was what fixed up my last few test failures. `p_wrapper` uses `edx.HtmlUtils`, so by calling that function to define a value, I was hitting the error before `init()` was run to set up the global values. Moving `unknown_p` to an evaluated-when-called function allowed `init()` to do it's thing before HmtlUtils is needed.